ui/src/tasks/components/TaskForm.tsx

```diff
@@ -145,6 +153,24 @@ export default class TaskForm extends PureComponent<Props, State> {
       </Form>
     )
   }
+  private get tokenDropdown(): JSX.Element {
```

Comment:
I can hear Brandon's voice in my head saying "if you have a getter that returns a component it should probably be it's own component"

Reply:
I agree. I created it to be separate component